pages/index.tsx

```diff
@@ -1,75 +1,347 @@
+            <h1 className="text-4xl font-extrabold tracking-tight text-center text-gray-100 sm:text-5xl md:text-6xl">
+              <span className="block xl:inline">Beautiful Media Discovery</span>
+              <span className="block text-indigo-500">
+                Simple Request Management
+              </span>
+            </h1>
+            <p className="mt-3 text-base text-center text-gray-400 sm:mt-5 sm:text-lg sm:max-w-xl sm:mx-auto md:mt-5 md:text-xl lg:mx-0">
+              Overseerr is a request management and media discovery tool built
+              to work with your existing Plex ecosystem.
+            </p>
+            <div className="w-full overflow-hidden bg-gray-800 rounded-md shadow-2xl md:rounded-xl">
+              <Image
+                src="/screenshots/screen_main.jpg"
+                width="1500"
+                height="948"
+                layout="responsive"
+              />
+            </div>
+          </div>
         </div>
-
-        <div className="mt-5 sm:mt-8 sm:flex sm:justify-center lg:justify-start">
-          <div className="rounded-md shadow">
-            <a
-              href="https://docs.overseerr.dev/getting-started/installation"
-              className="flex items-center justify-center w-full px-6 py-3 text-base font-medium text-white transition bg-indigo-600 border border-transparent rounded-md hover:bg-indigo-500 md:py-4 md:text-lg md:px-8"
-            >
-              <svg
-                className="w-6 h-6 mr-2"
-                fill="none"
-                stroke="currentColor"
-                viewBox="0 0 24 24"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  strokeLinecap="round"
-                  strokeLinejoin="round"
-                  strokeWidth={2}
-                  d="M4 16v1a3 3 0 003 3h10a3 3 0 003-3v-1m-4-4l-4 4m0 0l-4-4m4 4V4"
+      </div>
+      <div className="relative pb-32 overflow-hidden md:pt-16">
+        <div className="relative">
+          <div className="lg:mx-auto lg:max-w-7xl lg:px-8 lg:grid lg:grid-cols-2 lg:grid-flow-col-dense lg:gap-24">
+            <div className="max-w-xl px-4 mx-auto sm:px-6 lg:py-16 lg:max-w-none lg:mx-0 lg:px-0">
+              <div>
+                <div>
+                  <span className="flex items-center justify-center w-12 h-12 bg-indigo-600 rounded-md">
+                    <SparklesIcon
+                      className="w-6 h-6 text-white"
+                      aria-hidden="true"
+                    />
+                  </span>
+                </div>
+                <div className="mt-6">
+                  <h2 className="text-3xl font-extrabold tracking-tight text-gray-200">
+                    The best way to discover media
+                  </h2>
+                  <p className="mt-4 text-lg text-gray-400">
+                    Overseerr helps you find media you{' '}
+                    <em className="underline">want</em> to watch. With inline
```

Comment:
```suggestion
                    <em className="font-bold">want</em> to watch. With inline
```
Underline makes it look like a link, so perhaps bold here if you want extra emphasis?